packaging/src/main/assembly/src.xml

```diff
@@ -100,12 +100,26 @@
         <include>storage-api/**/*</include>
         <include>standalone-metastore/metastore-common/**/*</include>
         <include>standalone-metastore/metastore-server/**/*</include>
+        <include>standalone-metastore/metastore-tools/**/*</include>
+        <include>standalone-metastore/src/assembly/src.xml</include>
+        <include>standalone-metastore/pom.xml</include>
```

Comment:
Honestly, I am not sure what is the proper solution here. The `standalone-metastore` was separated in HIVE-17159 so that it gets its own release life-cycle. Since 2017, though I don't think this ever happened in practice. My concern here is the same with `storage-api`. We shouldn't be releasing the same code twice.

Reply:
these modules are not fully separated - meaning they are connected to the root pom.xml;
if we really want to separate them - we can put them into a new project...

but honestly I think these things were released separetly because there were no regular Hive releases....

Reply:
I don't know the full story behind these "independent" releases. If we want from now on to start releasing everything together that's fine with me. This could even help sanitize a bit the maven structure which is a bit messy at the moment. I just wanted to bring this up and be sure that what are doing is intentional.

Reply:
I think we should go what we have - and at least provide something which do compile in case someone want to do that...